docs/articles/test-workflows.md

```diff
@@ -374,7 +375,8 @@ spec:
           myParameter: param2
 ```
 
-> **Info:** Testkube uses the standard Kubernetes Cron format. See [Cron Format on Wikipedia](https://en.wikipedia.org/wiki/Cron) for details.
+> **Info:** Testkube uses the standard Kubernetes Cron format. See [Cron Format on Wikipedia](https://en.wikipedia.org/wiki/Cron) and [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) for details .
+
```

Comment:
please add info on which timezone will be used if no timezone is specified in the config